### PR TITLE
[keras/applications/mobilenet_v3.py] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

**Review by haifeng-jin — APPROVED** [review]
Thanks for the PR!

File: keras/applications/mobilenet_v3.py [hunk 1]
```diff
@@ -679,10 +679,10 @@ def preprocess_input(x, data_format=None):
 
     Args:
       x: A floating point `numpy.array` or a `tf.Tensor`.
-      data_format: Optional data format of the image tensor/array. Defaults to
-        None, in which case the global setting
-        `tf.keras.backend.image_data_format()` is used (unless you changed it,
-        it defaults to "channels_last").{mode}
+      data_format: Optional data format of the image tensor/array. `None` means
+        the global setting `tf.keras.backend.image_data_format()` is used
+        (unless you changed it, it uses "channels_last").{mode}.
```
Comment (qlzh727) [inline keras/applications/mobilenet_v3.py:684]: As I mentioned from other PR, the {mode} can be removed here. Please apply to any other PR as well.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

**Comment by haifeng-jin** [comment]
pending on author's change.

File: keras/applications/mobilenet_v3.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/applications/mobilenet_v3.py:684]: fixed

**Review by qlzh727 — APPROVED** [review]

**Review by haifeng-jin — APPROVED** [review]

**Pull request merged**